hpeOneView/resources/servers/id_pools_ipv4_ranges.py

```diff
@@ -156,5 +122,5 @@ def get_free_fragments(self, id_or_uri, count=-1, start=0):
         Returns:
             list: A list with the free fragments.
         """
-        uri = self._client.build_uri(id_or_uri) + "/free-fragments?start={0}&count={1}".format(start, count)
-        return self._client.get_collection(uri)
+        uri = uri + "/free-fragments?start={0}&count={1}".format(start, count)
+        return self._helper.get_collection(uri)
```

Comment:
```
uri = uri + "/free-fragments"
return self._helper.get_all(uri=uri)
```

Reply:
it works fine with get_collection() as well. 
Not sure if the requested changes are necessary to do ?